sendgrid_backend/mail.py

```diff
@@ -58,7 +58,42 @@ def __init__(self, *args, **kwargs):
 
         self.track_email = track_email
 
+        if hasattr(settings, "SENDGRID_ECHO_TO_STDOUT"):
```

Comment:
Shouldn't you also check that `SENDGRID_ECHO_TO_STDOUT` evaluates to a truthy value?

Reply:
...yes